server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/ExportAction.java

```diff
@@ -127,8 +130,9 @@ private void writeResponse(QualityProfileDto profile, @Nullable String exporterK
 
   private QualityProfileDto getProfile(@Nullable String name, String language) {
     try (DbSession dbSession = dbClient.openSession(false)) {
-      QualityProfileDto profile = name == null ? dbClient.qualityProfileDao().selectDefaultProfile(dbSession, language)
-        : dbClient.qualityProfileDao().selectByNameAndLanguage(name, language, dbSession);
+      OrganizationDto organization = qProfileWsSupport.getDefaultOrganization(dbSession);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getDefaultOrganization"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)